app/src/ui/components/Switch.jsx

```diff
@@ -0,0 +1,31 @@
+import styles from "./Switch.module.css";
+import clsx from "clsx";
+import { useState } from "react";
+
+const Switch = ({ onChange, defaultValue }) => {
+  const [isChecked, setIsChecked] = useState(defaultValue);
+  const handleCheckboxChange = event => {
+    setIsChecked(event.target.checked);
```

Comment:
just simply add onChange here and below in input->OnChange you can pass only 1 function. and reuse this one if needed with external callback always applied also